### DISCUSSION TITLE
IDK what title to use

### DISCUSSION THREAD

**Discussion by Charon-X-25** [BeardedBear/bearded-theme#96] [General]
okay so ... I just want the name of the dark theme in your screenshot , the one under "Classics , For those who don't like to be exotic and not suprised" , thx =D

**Comment by BeardedBear** [comment]
I think you mean "anthracite", I admit it's not very clear, I'll correct that when I update the image ^^

**Reply by Charon-X-25** [reply]
Thanks ^^
